test/spec/modules/richaudienceBidAdapter_spec.js

```diff
@@ -75,26 +75,19 @@ describe('Richaudience adapter tests', function () {
     user: {}
   }];
 
-  var DEFAULT_PARAMS_VIDEO_OUT_PARAMS = [{
+  var DEFAULT_PARAMS_BANNER_OUTSTREAM = [{
     adUnitCode: 'test-div',
     bidId: '2c7c8e9c900244',
     mediaTypes: {
-      video: {
-        context: 'outstream',
-        playerSize: [640, 480],
-        mimes: ['video/mp4']
+      banner: {
+        sizes: [[300, 250], [600, 300]]
       }
     },
     bidder: 'richaudience',
     params: {
       bidfloor: 0.5,
       pid: 'ADb1f40rmi',
-      supplyType: 'site',
-      player: {
-        init: 'close',
-        end: 'close',
-        skin: 'dark'
-      }
```

Comment:
This should also be removed from your markdown docs example

Reply:
the test has been removed